### PR TITLE
Update Collector Version

### PR DESCRIPTION
**Pull request by jgoldverg**
added in the updated collector 4.37.7
watch shazam amazing

### CODE REVIEW

**Comment by jgoldverg** [comment]
@daverted

File: overops_collector/app/collector.sh [hunk 1]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-set
 export TAKIPI_LISTEN_PORT=$PORT
 export TAKIPI_SERVER_NAME=$(echo $VCAP_APPLICATION | jq -r '.name')$INSTANCE_INDEX
 tar -xzvf takipi-latest.tar.gz
 sed -i "s @TAKIPI_BACKEND@ $TAKIPI_BASE_URL g" collector.properties
 cp collector.properties takipi
-nohup takipi/bin/takipi-service -nfd &
```
Comment (daverted) [inline overops_collector/app/collector.sh:8]: This should be in its own PR. Why did this change?

File: overops_collector/app/nohup.out [hunk 2]
```diff
@@ -0,0 +1 @@
+takipi/bin/takipi-service: takipi/bin/takipi-service: cannot execute binary file
```
Comment (daverted) [inline overops_collector/app/nohup.out]: this output file should not be in the repo

File: overops_collector/app/takipi/VERSION [hunk 3]
```diff
@@ -0,0 +1,2 @@
+
```
Comment (daverted) [inline overops_collector/app/takipi/VERSION]: The entire /takipi/ folder should not be in the repo. Only include the compressed tarball (.tar.gz file).

File: overops_collector/manifest.yml [hunk 4]
```diff
@@ -7,8 +7,8 @@ applications:
   random-route: true
   path: app
   env:  
-    TAKIPI_BASE_URL: https://backend.takipi.com/
-    TAKIPI_SECRET_KEY: S34504#uIniPX++SL8jpuTC#dO4Imyu8J61pxnqnZMkPYF7ekTp+vCq4bRUcl4aJiQk=#c84f
+    TAKIPI_BASE_URL: https://backend.overops.com/
+    TAKIPI_SECRET_KEY: S38797#0IBkissqY8b2aweQ#1vFP55KxTDmFqfaS3yD6haEKWdqfYgRGbNSuh17zD9c=#b1ae
```
Comment (daverted) [inline overops_collector/manifest.yml]: never include secrets in source control. this key should be removed.

File: t/META-INF/MANIFEST.MF [hunk 5]
```diff
@@ -0,0 +1,4 @@
+Manifest-Version: 1.0
+Created-By: 1.8.0_151 (Oracle Corporation)
+Main-Class: Hello
+
```
Comment (daverted) [inline t/META-INF/MANIFEST.MF]: t folder should be removed.

File: temp/takipi/VERSION [hunk 6]
```diff
@@ -0,0 +1,2 @@
+
+Release version: 4.37.7
```
Comment (daverted) [inline temp/takipi/VERSION]: temp folder should be removed

File: tile.yml [hunk 7]
```diff
@@ -12,7 +12,7 @@ packages:
   type: app
   manifest:
     memory: 1G
-    buildpack: java_buildpack_offline
+    buildpack: https://github.com/takipi-field/overops-tile.git
```
Comment (daverted) [inline tile.yml]: are you sure about this? seems out of scope of the PR, which is for updating the collector version

**Review by daverted — CHANGES_REQUESTED** [review]
make sure you keep the repo clean and limit changes to the scope of the PR

File: overops_collector/app/collector.sh [hunk 8]
```diff
@@ -1,8 +1,7 @@
 #!/bin/bash
-set
 export TAKIPI_LISTEN_PORT=$PORT
 export TAKIPI_SERVER_NAME=$(echo $VCAP_APPLICATION | jq -r '.name')$INSTANCE_INDEX
 tar -xzvf takipi-latest.tar.gz
```
Comment (daverted) [inline overops_collector/app/collector.sh]: This command isn't going to work, `takipi-latest.tar.gz` doesn't exist. The file was renamed `takipi-4.37.7.tar.gz` in this commit.

**Review by daverted — CHANGES_REQUESTED** [review]
Double check `collector.sh`

**Review by daverted — APPROVED** [review]
👍
